idem_azurerm/exec/azurerm/network/load_balancer.py

```diff
@@ -207,10 +207,12 @@ async def create_or_update(hub, name, resource_group, **kwargs):
                     }
             # Use Subnet name to link to the ID of an existing Subnet
             elif "subnet" in kwargs["frontend_ip_configurations"][idx]:
-                vnets = virtual_networks_list(resource_group=resource_group, **kwargs)
+                vnets = await hub.exec.azurerm.network.virtual_networks.list(
+                    resource_group=resource_group, **kwargs
+                )
                 if "error" not in vnets:
                     for vnet in vnets:
-                        subnets = subnets_list(
+                        subnets = await hub.exec.azurerm.network.subnets.list(
```

Comment:
I believe that this should be `hub.exec.azurerm.network.virtual_network.subnets_list`